mappings/com/mojang/blaze3d/systems/RenderSystem.mapping

```diff
@@ -1,169 +1,376 @@
+	METHOD initBackendSystem initBackendSystem ()Ljava/util/function/LongSupplier;
+	METHOD initGameThread initGameThread (Z)V
 		ARG 0 assertNotRenderThread
-	METHOD initRenderer (IZ)V
+	METHOD initRenderThread initRenderThread ()V
+	METHOD initRenderer initRenderer (IZ)V
 		ARG 0 debugVerbosity
 		ARG 1 debugSync
-	METHOD limitDisplayFPS (I)V
+	METHOD isInInitPhase isInInitPhase ()Z
+	METHOD isOnGameThread isOnGameThread ()Z
+	METHOD isOnRenderThread isOnRenderThread ()Z
+	METHOD isOnRenderThreadOrInit isOnRenderThreadOrInit ()Z
+	METHOD lambda$static$0 (Lit/unimi/dsi/fastutil/ints/IntConsumer;I)V
+		ARG 0 indexBuilder
+		ARG 1 primitiveId
+	METHOD lambda$static$1 (Lit/unimi/dsi/fastutil/ints/IntConsumer;I)V
+		ARG 0 indexBuilder
+		ARG 1 primitiveId
+	METHOD limitDisplayFPS limitDisplayFPS (I)V
 		ARG 0 fps
-	METHOD lineWidth (F)V
+	METHOD lineWidth lineWidth (F)V
 		ARG 0 width
-	METHOD logicOp (Lcom/mojang/blaze3d/platform/GlStateManager$C_hfcoodvl;)V
+	METHOD logicOp logicOp (Lcom/mojang/blaze3d/platform/GlStateManager$C_hfcoodvl;)V
 		ARG 0 op
-	METHOD pixelStore (II)V
+	METHOD maxSupportedTextureSize maxSupportedTextureSize ()I
+	METHOD pixelStore pixelStore (II)V
 		ARG 0 pname
 		ARG 1 param
-	METHOD polygonOffset (FF)V
+	METHOD polygonMode polygonMode (II)V
+		ARG 0 face
+		ARG 1 mode
+	METHOD polygonOffset polygonOffset (FF)V
 		ARG 0 factor
 		ARG 1 units
-	METHOD readPixels (IIIIIILjava/nio/ByteBuffer;)V
+	METHOD readPixels readPixels (IIIIIILjava/nio/ByteBuffer;)V
 		ARG 0 x
 		ARG 1 y
 		ARG 2 width
 		ARG 3 height
 		ARG 4 format
 		ARG 5 type
 		ARG 6 pixels
-	METHOD recordRenderCall (Lnet/minecraft/unmapped/C_ycxumudt;)V
+	METHOD recordRenderCall recordRenderCall (Lnet/minecraft/unmapped/C_ycxumudt;)V
 		ARG 0 renderCall
-	METHOD renderCrosshair (I)V
+	METHOD renderCrosshair renderCrosshair (I)V
 		ARG 0 size
-	METHOD setErrorCallback (Lorg/lwjgl/glfw/GLFWErrorCallbackI;)V
+	METHOD renderThreadTesselator renderThreadTesselator ()Lnet/minecraft/unmapped/C_hiiunyvh;
+	METHOD replayQueue replayQueue ()V
+	METHOD resetTextureMatrix resetTextureMatrix ()V
+	METHOD restoreProjectionMatrix restoreProjectionMatrix ()V
+	METHOD runAsFancy runAsFancy (Ljava/lang/Runnable;)V
+		ARG 0 runnable
+	METHOD setErrorCallback setErrorCallback (Lorg/lwjgl/glfw/GLFWErrorCallbackI;)V
 		ARG 0 callback
-	METHOD setupDefaultState (IIII)V
+	METHOD setInverseViewRotationMatrix setInverseViewRotationMatrix (Lnet/minecraft/unmapped/C_crutkuff;)V
+		ARG 0 inverseViewRotationMatrix
+	METHOD setProjectionMatrix setProjectionMatrix (Lnet/minecraft/unmapped/C_gxvddjib;)V
+		ARG 0 projectionMatrix
+	METHOD setShader setShader (Ljava/util/function/Supplier;)V
+		ARG 0 shader
+	METHOD setShaderColor setShaderColor (FFFF)V
+		ARG 0 r
+		ARG 1 g
+		ARG 2 b
+		ARG 3 a
+	METHOD setShaderFogColor setShaderFogColor (FFF)V
+		ARG 0 r
+		ARG 1 g
+		ARG 2 b
+	METHOD setShaderFogColor setShaderFogColor (FFFF)V
+		ARG 0 r
+		ARG 1 g
+		ARG 2 b
+		ARG 3 a
```

Comment:
These are inconsistent with some of the other methods here, e.g. `colorMask`, `clearColor`
```suggestion
	METHOD setShaderColor setShaderColor (FFFF)V
		ARG 0 red
		ARG 1 green
		ARG 2 blue
		ARG 3 alpha
	METHOD setShaderFogColor setShaderFogColor (FFF)V
		ARG 0 red
		ARG 1 green
		ARG 2 blue
	METHOD setShaderFogColor setShaderFogColor (FFFF)V
		ARG 0 red
		ARG 1 green
		ARG 2 blue
		ARG 3 alpha
```